wrappers/vcx-napi-rs/index.d.ts

```diff
@@ -97,6 +97,7 @@ export function createServiceV2(targetDid: string, routingKeys: Array<string>, e
 export function getServiceFromLedger(targetDid: string): Promise<string>
 export function getAttrFromLedger(targetDid: string, attr: string): Promise<string>
 export function clearAttrFromLedger(did: string, attrib: string): Promise<string>
+export function writeEndorserDid(submitterDid: string, targetDid: string, targetVk: string, alias?: string | undefined | null): Promise<string>
```

Comment:
I assume this was generated by napi-rs, but I would have thought that just `alias?: string | null` is equivalent...

Reply:
I think you are right, but since it's auto-generated, I won't modify it